scripts/build-testing-sdk.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * This script bundles some scripts used for testing:
+ * - amd loader
+ * - polyfills
+ *   + fetch
+ *   + promises
+ *   + bace64 encoding
+ * - then deps for the testing framework
+ */
+const fs = require('fs');
+
+const sourceModules = getSourceFromModules([
+  // umd module loader
+  'loader.js',
+  // original testing sdk deps
+  'route-recognizer',
+  'fake-xml-http-request',
+  'pretender'
+]);
+
+const outSource = `
+${sourceModules[0]}
+(function (self) {
+;self.fetch = null;
+${sourceModules.slice(1).join('\n')}
+}(window));
+`;
+
+console.log(outSource);
+
+function getSourceFromModules(modules) {
+
+  return modules
```

Comment:
Is there a reason to not call in one line like `modules.map(...)`? It's kind of weird to me to have the line break there. I'm curious on your opinion though.